tools/code/pmtech_editor/pmtech_editor.cpp

```diff
@@ -129,7 +129,6 @@ namespace
         // create the main scene and camera
         main_scene = put::ecs::create_scene("main_scene");
         camera_create_perspective(&main_camera, 60.0f, put::k_use_window_aspect, 0.1f, 1000.0f);
-
```

Comment:
please re-instate this space, again subtle but important, commented blocks of code with lines without spaces are there to indicate groups of functionality.

this file does not need to change. it can remain as it was prior the the PR